Homestead/readme.md

```diff
@@ -119,11 +119,6 @@ Finally, make one change to the `after.sh` file:
 # Use "polling" rather than unsupported filesystem events for tools like Webpack:
 echo 'export CHOKIDAR_USEPOLLING=true' >> /home/vagrant/.bash_profile
 
-# Switch to PHP 7.2 for the default CLI.
-sudo update-alternatives --set php /usr/bin/php7.2
-sudo update-alternatives --set php-config /usr/bin/php-config7.2
-sudo update-alternatives --set phpize /usr/bin/phpize7.2
-
```

Comment:
Does the default CLI now automatically use 7.3? No need to tell it to use the updated alternative?

Reply:
Yep, that's the default since it's the latest stable version.